omnipaxos/src/omni_paxos.rs

```diff
@@ -295,15 +311,13 @@ where
     }
 }
 
-/// An error returning the proposal that was failed due to that the current configuration is stopped.
+/// An error indicating a failed proposal due to the current configuration being already stopped
+/// or due to an invalid proposed configuration.
 #[allow(missing_docs)]
 #[derive(Debug)]
-pub enum ProposeErr<T>
-where
-    T: Entry,
-{
-    Normal(T),
-    Reconfiguration(ClusterConfig),
+pub enum ProposeErr {
+    Stopped,
+    Reconfiguration(ConfigError),
 }
```

Comment:
I'm thinking that we should get rid of this enum because they are for two different things.

`Stopped` is for when calling `append() `while `Reconfiguration` is for calling `reconfigure()`.

I also suggest to rename `Stopped` to `PendingReconfiguration`, since it might not actually be stopped, but only pending (the StopSign could even later be dropped).

Reply:
Probably this is a result my bad naming also for the `stopped()` function. That should be renamed to `pending_reconfiguration()`